src/launcher/features/localAppInstall/DropZoneForLocalApps.test.tsx

```diff
@@ -19,6 +19,12 @@ const { appExists, failureReadingFile, successfulInstall } =
 
 jest.mock('../../../ipc/apps');
 
+jest.mock('electron', () => ({
+    webUtils: {
+        getPathForFile: jest.fn((file: { path: string }) => file.path), // treat "file" as "path" for the tests
+    },
+}));
```

Comment:
Not sure this is a good way to fix tests as it is testing with an old behavior under the hood. Please reject if you think test suits should be updated instead